pages/txs.tsx

```diff
@@ -39,70 +28,91 @@ const TxList = () => {
-                        {t('to')}
-                      </TableCell>
-                      <TableCell sx={{ display: { xs: 'table-cell', md: 'none' } }} component="th">
-                        {t('transfer')}
-                      </TableCell>
-                      <TableCell component="th" sx={{ whiteSpace: 'nowrap', textTransform: 'none' }}>{`${t(
-                        'value',
-                      )} (${PCKB_SYMBOL})`}</TableCell>
-                      <TableCell component="th">{t('type')}</TableCell>
-                    </TableRow>
-                  </TableHead>
-                  <TableBody>
-                    {Array.from({ length: +page_size }).map((_, idx) => (
-                      <TableRow key={idx}>
-                        <TableCell colSpan={7} sx={{ display: { xs: 'table-cell', md: 'none' } }}>
-                          <Skeleton animation="wave" height={25} />
-                        </TableCell>
-                        <TableCell colSpan={8} sx={{ display: { xs: 'none', md: 'table-cell' } }}>
-                          <Skeleton animation="wave" height={25} />
-                        </TableCell>
-                      </TableRow>
-                    ))}
-                  </TableBody>
-                </Table>
-              </TableContainer>
-              <Stack direction="row" justifyContent="space-between" alignItems="center" py={1}>
-                <Skeleton animation="wave" width={50} height={24} />
-                <Skeleton animation="wave" width={50} height={24} />
-              </Stack>
-            </Box>
-          </Paper>
+          <Box
+            sx={{
+              borderRadius: {
+                xs: 2,
+                sm: 4,
+              },
+              border: {
+                xs: '0.5px solid #F0F0F0',
+                md: '1px solid #F0F0F0',
+              },
+              pt: { xs: 1.5, md: 2 },
+              pb: 2,
+              mt: { xs: 2, md: 3 },
+              bgcolor: '#fff',
+            }}
+          >
+            <Stack
+              direction="row"
+              flexWrap="wrap"
+              justifyContent="space-between"
+              alignItems="center"
+              mb={{ xs: 1.5, md: 2 }}
+              px={{ xs: 1.5, md: 3 }}
+            >
+              <Skeleton animation="wave" width={50} height={24} />
+              <Skeleton animation="wave" width={50} height={24} />
+            </Stack>
+            <TableContainer>
+              <Table>
+                <thead style={{ textTransform: 'capitalize', fontSize: isMobile ? 12 : 14 }}>
+                  <tr>
+                    <th>{t('txHash')}</th>
+                    <th>
+                      <Stack direction="row" alignItems="center" whiteSpace="nowrap">
+                        {t('block')}
+                      </Stack>
+                    </th>
+                    <th>{t('age')} </th>
+                    <th style={{ display: isMobile ? 'none' : 'table-cell' }}>{t('from')}</th>
```

Comment:
The design draft didn't change columns for mobile
![image](https://user-images.githubusercontent.com/7271329/182580361-7c6d76ca-cd29-482f-ab03-70de95e21e85.png)

Reply:
My fault, this is the skeleton for loading